src/app/static/src/tests/components/uploadInputs/uploadInputs.test.ts

```diff
@@ -90,6 +91,66 @@ describe("UploadInputs upload component", () => {
         expect(wrapper.findAll(ManageFile).at(0).props().accept).toBe("PJNZ,pjnz,.pjnz,.PJNZ,.zip,zip,ZIP,.ZIP");
     });
 
+    it("pjnz upload does not show asterisk for required field when on data exploration mode", () => {
+        const store = createSut();
+        expectFileIsNotRequired(store, 0)
+    });
+
+    it("pjnz upload shows asterisk for required field when not on data exploration mode", () => {
+        const store = createSut({}, {}, {}, false);
+        expectFileIsRequired(store, 0)
+    });
+
+    it("area file upload shows asterisk for required field when on data exploration mode", () => {
+        const store = createSut();
+        expectFileIsRequired(store, 1)
+    });
+
+    it("area file upload shows asterisk for required field when not on data exploration mode", () => {
+        const store = createSut({}, {}, {}, false);
+        expectFileIsRequired(store, 1)
+    });
+
+    it("population upload does not show asterisk when on data exploration mode", () => {
+        const store = createSut();
+        expectFileIsNotRequired(store, 2)
+    });
+
+    it("population upload shows asterisk for required field when not on data exploration mode", () => {
+        const store = createSut({}, {}, {}, false);
+        expectFileIsRequired(store, 2)
+    });
+
+    it("survey upload does not show asterisk when on data exploration mode", () => {
+        const store = createSut();
+        expectFileIsNotRequired(store, 3)
+    });
+
+    it("survey upload shows asterisk for required field when not on data exploration mode", () => {
+        const store = createSut({}, {}, {}, false);
+        expectFileIsRequired(store, 3)
+    });
+
+    it("ART upload does not show asterisk when on data exploration mode", () => {
+        const store = createSut();
+        expectFileIsNotRequired(store, 4)
+    });
+
+    it("ART upload does not show asterisk when not on data exploration mode", () => {
+        const store = createSut();
+        expectFileIsNotRequired(store, 4)
+    });
+
+    it("ANC upload does not show asterisk when on data exploration mode", () => {
+        const store = createSut();
+        expectFileIsNotRequired(store, 4)
+    });
+
+    it("ANC upload does not show asterisk when not on data exploration mode", () => {
+        const store = createSut();
+        expectFileIsNotRequired(store, 4)
+    });
```

Comment:
Looks like you're testing both ART and ANC files with index 4. This suggests the expect methods taking an index param might be a bit prone to error e.g. files might get reordered potentially - maybe those helpers could find the ManageFile components by label attribute instead, and take that as a parameter?